source/administration/console/security-and-access.rst

```diff
@@ -145,6 +145,8 @@ Configuring an external IDP enables Single-Sign On workflows, where applications
 Use the the screens in this section to view, add, or edit OIDC configurations for the deployment.
 MinIO supports any number of active OIDC configurations.
 
+.. _minio-console-admin-identity-ldap:
+
 LDAP
```

Comment:
This Console page is fairly vague, contemplating what to add about current LDAP configuration that isn't too out of character with the rest of it.

See also https://github.com/minio/console/issues/2967

Reply:
The Console steps in the main LDAP page is partly an include, which could be used here as well.

Reply:
Keeping the anchor, but leaving this page as it is for now.